webapp/web/.gcloudignore

```diff
@@ -1,28 +1,35 @@
-# Don't upload test data, package config, etc. to AppEngine.
```

Comment:
I think this line is still useful?

Reply:
Hmm, yeah. I broke it into categories, but I should keep a summary line for this file.